Sources/Apodini/Semantic Model Builder/GraphQL/GraphQLSchemaBuilder.swift

```diff
@@ -0,0 +1,248 @@
+//
+// Created by Sadik Ekin Ozbay on 06.01.21.
+//
+
+@_implementationOnly import Vapor
+import GraphQL
+
+
+func graphqlTypeMap(with type: Codable.Type) -> GraphQLScalarType {
+    if (type == String.self) {
+        return GraphQLString
+    } else if (type == Int.self || type == UInt.self) {
+        return GraphQLInt
+    } else if (type == Float.self) {
+        return GraphQLFloat
+    } else if (type == Bool.self) {
+        return GraphQLBoolean
+    }
+    return GraphQLString
+
+}
+
+struct GraphQLRequest: ExporterRequest {
+    var source: Any
+    var args: Map
+    var context: Any
+    var info: GraphQLResolveInfo
+}
+
+
+class GraphQLSchemaBuilder {
+    private var tree = [String: Set<String>]()
+    private var leafContext = [String: AnyConnectionContext<GraphQLInterfaceExporter>]()
+    private var hasIncomingEdge = Set<String>()
+
+
+    // GraphQL Related values
+    private var types = [String: GraphQLObjectType]()
+    private var fields = [String: GraphQLField]()
+    private var args = [String: [String: GraphQLArgument]]()
+    private var responseTypeTree = [String: Node<EnrichedInfo>]()
+
+    private func typeToGraphQL(type: Any.Type) -> GraphQLType? {
+        if (type == String.self) {
+            return GraphQLString
+        }
+        if (type == Int.self || type == UInt.self) {
+            return GraphQLInt
+        }
+        if (type == Bool.self) {
+            return GraphQLBoolean
+        }
+        if (type == Float.self) {
+            return GraphQLFloat
+        }
+        return nil
+    }
+
+    private func responseTypeHandler(for responseTypeHead: Node<EnrichedInfo>) -> GraphQLOutputType {
+        let typeValTemp = self.typeToGraphQL(type: responseTypeHead.value.typeInfo.type)
+
+        if let typeVal = typeValTemp {
+            return typeVal as! GraphQLOutputType
+        }
```

Comment:
I think you can remove the force cast here:
```suggestion
        if let typeVal = typeValTemp as? GraphQLOutputType {
            return typeVal
        }
```